packages/support/lib/image-util.js

```diff
@@ -1,13 +1,126 @@
 import _ from 'lodash';
 import Jimp from 'jimp';
-import {Buffer} from 'buffer';
 import {PNG} from 'pngjs';
 import B from 'bluebird';
 
 const BYTES_IN_PIXEL_BLOCK = 4;
 const SCANLINE_FILTER_METHOD = 4;
 const {MIME_JPEG, MIME_PNG, MIME_BMP} = Jimp;
 
+/**
+ * Wrapper around a {@linkcode Jimp} image object.
+ */
+
+export class AppiumImage {
+  /** @type {Jimp} */
+  #image;
+
+  /**
+   * @param {Jimp} image
+   */
+  constructor(image) {
+    this.#image = image;
+  }
+
+  toString() {
+    return `<AppiumImage <${this.#image.toString()}>`;
+  }
+
+  get bitmap() {
+    return this.#image.bitmap;
+  }
+
+  /**
+   * @param {string} mime
+   * @returns {Promise<Buffer>}
+   */
+  async getBuffer(mime) {
+    return await this.#image.getBufferAsync(mime);
+  }
+
+  /**
+   * @param {string} path
+   * @returns {Promise<AppiumImage>}
+   */
+  async write(path) {
+    await this.#image.writeAsync(path);
+    return this;
+  }
+
+  /**
+   * @param {string} mime
+   * @returns {Promise<string>}
+   */
+  async getBase64(mime) {
+    return await this.#image.getBase64Async(mime);
+  }
+
+  /**
+   *
+   * @param {number} width
+   * @param {number} height
+   * @returns {AppiumImage}
+   */
+  resize(width, height) {
+    this.#image.resize(width, height);
+    return this;
+  }
+
+  /**
+   * @param {number} width
+   * @param {number} height
+   * @returns {AppiumImage}
+   */
+  scaleToFit(width, height) {
+    this.#image.scaleToFit(width, height);
+    return this;
+  }
+
+  /**
+   *
+   * @param {string|Buffer} strOrBuf
+   * @returns {Promise<AppiumImage>}
+   */
+  static async from(strOrBuf) {
+    return await (_.isString(strOrBuf)
+      ? AppiumImage.fromString(strOrBuf)
+      : AppiumImage.fromBuffer(strOrBuf));
+  }
+
+  /**
+   *
+   * @param {string} str
```

Comment:
I assume it should explicitly say that base64-encoded image payload is expected. Also it would make sense to list supported image formats

Reply:
will do